themes/finna2/less/finna/feed.less

```diff
@@ -510,6 +512,15 @@ finna-feed {
                 display: none;
             }
         }
+        @media (max-width: @screen-md) {
+            bottom: -5px;
```

Comment:
Mietin saisiko autoplay-painikeeen ja pallot keskitettyä pelkästään flexillä? Ei tarvis siten käyttää pikselin tarkkaa keskitystä.

Reply:
Kannatan. Käytetään flexiä vaan kaikessa tarvittavassa ennemmin kuin asetellaan pikseleittäin.